tests/stdlib/tmath.nim

```diff
@@ -1,4 +1,15 @@
-import math, random
+discard """
+  action: run
+  output: '''[Suite] random int
```

Comment:
Yes, that is why I came up with the idea of making the output check optional. And it is also what napalu had to do to fix #5472 in f61bb70.